main.cpp

```diff
@@ -18,7 +19,15 @@ int main(int argc, char *argv[])
     ExitHandler::setApplication(&app);
     ExitHandler::setup();
 
+    // Initializating referee core
+    RefereeCore *refereeCore = new RefereeCore();
```

Comment:
The **RefereeCore** class will be responsible for creating all pointers to the program modules and their instantiation in the World class, which will subsequently maintain these modules.